synthesizer/event.cpp

```diff
@@ -391,8 +394,18 @@ void EventMap::fixupMIDI()
 
       auto it = begin();
       while (it != end()) {
+            NPlayEvent event = it->second;
```

Comment:
Isn't it possible to avoid code duplication? Extract a method?

Reply:
Yes, I agree that it is better to avoid code duplication. I just pushed a change to this branch that does what you suggest.